packages/react/src/themes/teams-high-contrast/components/Attachment/attachmentVariables.ts

```diff
@@ -6,7 +6,7 @@ export default (siteVariables: any): Partial<AttachmentVariables> => ({
   backgroundColorHover: siteVariables.accessibleYellow,
   textColor: siteVariables.colors.white,
   textColorHover: siteVariables.colors.black,
-  boxShadow: undefined,
+  boxShadow: null,
```

Comment:
Try using `unset` CSS value here instead of the `null` concept.  Hopefully, we can stick with built-in CSS functionality here.

Reply:
Looking at the rest of the code, specifically variables, if the `null` concept works everywhere (styles/variables) for "removing" values or "unsetting" them, then so be it 👍 

Let's wait to hear some evidence from users.